cellmlmanip/model.py

```diff
@@ -410,10 +410,14 @@ def get_equation_graph(self, refresh=False) -> nx.DiGraph:
         self.graph = graph
         return graph
 
-    def get_equations_for(self, symbols):
-        """Get all equations for given collection of symbols"""
+    def get_equations_for(self, symbols, lexicographical_sort=True):
+        """Get all equations for given collection of symbols
```

Comment:
Docstrings should have the format

    """
    One-line description

    Longer description, which may include a list of arguments (in a syntax that's unfortunately project-specific).
    The language for docstrings is "reStructuredText", in which you indicate variable names (and other code) with two backticks, like ``this``
    """